packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -46,6 +46,10 @@ import 'ticker_provider.dart';
 /// if widgets in an overlay entry with [maintainState] set to true repeatedly
 /// call [State.setState], the user's battery will be drained unnecessarily.
 ///
+/// If this entry is declared to be a [semanticsBarrier], then entries below
+/// this one will not be included in the semantic tree, e.i. those entries will
+/// be unreachable via accessibility means.
```

Comment:
"via accessibility means" -> "to accessibility tools" or "via accessibility user agents" or some such.